system/main/home.php

```diff
@@ -126,6 +126,7 @@ function checkFilesDir() {
     $handle = fopen($files_dir . "/.htaccess", 'w+');
     if ($handle) {
       fwrite($handle, "Deny from all\n");
+      fwrite($handle, "# IMPORTANT, prevents directory content listing!\nOptions -Indexes\n");
```

Comment:
Was my first thought, too.
However, if the file exists and the update is running, then the file must be old per definition.
On the other hand, if the file doesn't exist, than it will not be created here.
So, I don't see any way how this line ends up in the file twice, agree?